infrastructure/labelmaker/labelmaker.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+    labelmaker.py is a tool that reads key:value pairs from a json file, and
+    labels the running instance and all attached drives accordingly. It
+    is designed to run on boot in a startup-script or userdata.
+"""
+
+from __future__ import print_function
+
+import sys
+import json
+from operator import itemgetter
+import requests
+from googleapiclient import discovery
+from oauth2client.client import GoogleCredentials
+
+def label(self_link, access_token, data):
```

Comment:
should have two blank lines for top level definitions: https://www.python.org/dev/peps/pep-0008/#blank-lines